surfaceflinger.te

```diff
@@ -1 +1,3 @@
 rw_dir_file(surfaceflinger, sysfs_graphics)
+
+allow surfaceflinger sysfs:file w_file_perms;
```

Comment:
We should determine which sysfs and create a label for it, this allows to write to **all** sysfs